org.eclipse.winery.workflowmodeler/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "workflow-modeler",
+  "version": "1.0.0",
+  "description": "A Designer For workflow",
+  "scripts": {
+    "start": "webpack-dev-server --inline --progress",
+    "test": "karma start",
+    "build": "rimraf dist && webpack --config config/webpack.prod.js --progress --profile --bail",
+    "postinstall": "typings install",
+    "mock": "node mock/server.js"
+  },
+  "licenses": [
+    {
+      "type": "MIT",
+      "url": "https://github.com/angular/angular.io/blob/master/LICENSE"
+    }
+  ],
+  "dependencies": {
+    "@angular/common": "~2.1.0",
+    "@angular/compiler": "~2.1.0",
+    "@angular/core": "~2.1.0",
+    "@angular/forms": "~2.1.0",
+    "@angular/http": "~2.1.0",
+    "@angular/platform-browser": "~2.1.0",
+    "@angular/platform-browser-dynamic": "~2.1.0",
+    "@angular/router": "~3.1.0",
+    "@ng-bootstrap/ng-bootstrap": "^1.0.0-alpha.9",
+    "angular-in-memory-web-api": "0.1.5",
+    "angular2-uuid": "^1.1.0",
+    "bootstrap": "^4.0.0-alpha.6",
+    "core-js": "^2.4.1",
+    "font-awesome": "^4.7.0",
+    "moment": "^2.15.2",
+    "rxjs": "5.0.0-beta.12",
+    "zone.js": "^0.6.25"
+  },
+  "devDependencies": {
+    "angular2-template-loader": "^0.4.0",
```

Comment:
In addition, please try to use the latest stable releases of your dependencies and devDependencies (you could use the npm-check-update tool).